receiver/factorytemplate/factory_template.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package factorytemplate allows easy construction of factories for receivers.
+// It requires the consumer to provide only functions to create the default
+// configuration and how to create the receiver from that configuration.
+package factorytemplate
```

Comment:
nit: Let's may a convention that the main file in a package should have the same name, in this case factorytemplate.go

Reply:
*make a convention

Reply:
ok, renaming it.